### PR TITLE
[ADD] add documentation about `Model._read_group`

### PR DESCRIPTION
**Pull request by ryv-odoo**
task-3001022

odoo/odoo#110737
odoo/enterprise#38639

### CODE REVIEW

**Review by rco-odoo — APPROVED** [review]
@robodoo r+

**Comment by rco-odoo** [comment]
@AntoineVDV can you review it?  Thanks :heart:

File: content/developer/reference/backend/orm.rst [hunk 1]
```diff
@@ -859,6 +859,8 @@ Search/Read
 
 .. automethod:: Model.read
 
+.. automethod:: Model._read_group
```
Comment (AntoineVDV) [inline content/developer/reference/backend/orm.rst:862]: See https://github.com/odoo/odoo/pull/110737#pullrequestreview-1391805363 for the review of the docstring.

**Review by AntoineVDV — COMMENTED** [review]
@robodoo delegate+

Feel free to re-ask for review but it should be good after the comments below are resolved :)

**Comment by ryv-odoo** [comment]
@robodoo r+

**Pull request closed without merging**